src/did/resolver.test.ts

```diff
@@ -128,16 +138,16 @@ describe("getProviderConfig", () => {
 
   it("should use env variables for provider config properties for didResolver", () => {
     process.env.PROVIDER_ENDPOINT_TYPE = "alchemy";
-    process.env.PROVIDER_NETWORK = "goerli";
+    process.env.PROVIDER_NETWORK = "sepolia";
 
     expect(getProviderConfig()).toEqual({
-      networks: [{ name: "goerli", rpcUrl: "https://eth-goerli.alchemyapi.io/v2/_gg7wSSi0KMBsdKnGVfHDueq6xMB9EkC" }],
+      networks: [{ name: "sepolia", rpcUrl: "https://eth-sepolia.g.alchemy.com/v2/" }],
```

Comment:
No default API key used here because we are using a workaround with manual URL (`JsonRPCProvider`) for ethers v5 to call Sepolia. Default API key will apply in ethers v6 when we use `AlchemyProvider` which supports Sepolia